proto/types.proto

```diff
@@ -12,14 +12,59 @@ message ExtraData {
 	google.protobuf.Any RuncOptions = 2;
 }
 
+// Message to specify the various Firecracker VM configurations for a task
+message FirecrackerConfig {
+	google.protobuf.Any RuncOptions = 1; // Specifies Runc options that need to relayed to the CreateTask request
+	repeated FirecrackerNetworkInterface NetworkInterfaces = 2; // Specifies the networking configuration for a VM
+	string KernelImagePath = 3; // Specifies the file path where the kernel image is located
+	string KernelArgs = 4; // Specifies the commandline arguments that should be passed to the kernel
```

Comment:
Should this be `repeated string`?  (I haven't thought about this in depth)

Reply:
We may at some point end up wanting a more featureful kernel commandline constructor. But I don't really want to go there if we don't have to. IMO leaving it as a single opaque string is probably going to avoid the most madness.